CMake Error at CMakeLists.txt:220 (find_package):
  Could not find a package configuration file provided by "SPIRV-Headers"
  with any of the following names:

    SPIRV-HeadersConfig.cmake
    spirv-headers-config.cmake

  Add the installation prefix of "SPIRV-Headers" to CMAKE_PREFIX_PATH or set
  "SPIRV-Headers_DIR" to a directory containing one of the above files.  If
  "SPIRV-Headers" provides a separate development package or SDK, be sure it
  has been installed.

--- CMakeLists.txt.orig	2022-10-28 03:54:50 UTC
+++ CMakeLists.txt
@@ -95,6 +95,8 @@ if (SPIRV_HEADERS_INSTALL_DIR)
 endif()
 if (SPIRV_HEADERS_INSTALL_DIR)
     list(APPEND CMAKE_PREFIX_PATH ${SPIRV_HEADERS_INSTALL_DIR})
+    add_library(SPIRV-Headers::SPIRV-Headers INTERFACE IMPORTED)
+    target_include_directories(SPIRV-Headers::SPIRV-Headers INTERFACE ${SPIRV_HEADERS_INSTALL_DIR}/include)
 endif()
 if (SPIRV_TOOLS_INSTALL_DIR)
     list(APPEND CMAKE_PREFIX_PATH ${SPIRV_TOOLS_INSTALL_DIR})
@@ -213,7 +215,9 @@ if(BUILD_LAYERS OR BUILD_TESTS)
 endif()
 
 if(BUILD_LAYERS OR BUILD_TESTS)
+  if (NOT TARGET SPIRV-Headers::SPIRV-Headers)
     find_package(SPIRV-Headers REQUIRED CONFIG QUIET)
+  endif()
 
     include(VVLGenerateSourceCode)
 
